packages/lesswrong/components/notifications/NotificationsItem.tsx

```diff
@@ -126,6 +126,14 @@ const NotificationsItem = ({notification, lastNotificationsCheck, currentUser, c
   
   return (
     <span {...eventHandlers}>
+      <LWPopper
```

Comment:
Move the popper outside of the a-tag to fix the bug where you couldn't click on links in the hoverover.